src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -186,15 +186,18 @@ export class WizardModal extends Modal {
 		if (validate && !await this.validateNavigation(index)) {
 			return;
 		}
+
+		let dialogPaneToShow: DialogPane | undefined = undefined;
 		this._dialogPanes.forEach((dialogPane, page) => {
 			if (page === pageToShow) {
+				dialogPaneToShow = dialogPane;
 				dialogPane.show(focus);
 			} else {
 				dialogPane.hide();
 			}
 		});
 
-		status(localize('wizardPageChangedMessage', "Step {0}: {1}", index + 1, pageToShow.title));
+		status(localize('wizardPageChangedMessage', "{0} {1}", dialogPaneToShow.pageNumberDisplayText, dialogPaneToShow.title));
```

Comment:
Combining already localized strings like this doesn't need to use localize. The localizers have no context and realistically this is going to already be broken for non-LTR languages so if we really wanted to make it work for those scenarios we'd want to do something different anyways.